v3/src/data-interactive/handlers/log-message-handler.ts

```diff
@@ -1,8 +1,28 @@
+import { LoggableObject, LoggableValue, logMessageWithReplacement } from "../../lib/log-message"
 import { registerDIHandler } from "../data-interactive-handler"
-import { DIHandler, diNotImplementedYet } from "../data-interactive-types"
+import { DIHandler, DILogMessage, DIResources, DIValues,  } from "../data-interactive-types"
 
 export const diLogMessageHandler: DIHandler = {
-  notify: diNotImplementedYet
+  notify(resources: DIResources, values?: DIValues) {
+    const webviewModel = resources.interactiveFrame?.content
+    if (values) {
```

Comment:
Did you double check how this works in v2? If not, you should definitely try using the API tester plugin to try this out. I seem to remember the API handler sending an empty log message when no values are included, but maybe I'm misremembering. If you do want to send something even if no values are specified, you could replace the following line with this:
`const { formatStr, replaceArgs } = (values ?? {}) as DILogMessage`

Reply:
We tried no `replaceArgs`, and that worked.
V2 returns this error when there is no `values`:
```{
  "success": false,
  "values": {
    "error": "TypeError: Cannot read properties of undefined (reading 'topic')"
  }
}```